_docs/_latest/setup/manual-deployment.md

```diff
@@ -0,0 +1,162 @@
+if you already created the instance, you can [update the service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes).
+
+### Install the Forseti Server
+
+To install the Forseti Server, follow the steps below:
+
+1. SSH into the server VM
+1. Run the following command to become an Ubuntu user:
+    ```bash
+    sudo su ubuntu
+    ```
+1. `git clone` from the [forseti repo](https://github.com/GoogleCloudPlatform/forseti-security), and check out the [latest release](https://github.com/GoogleCloudPlatform/forseti-security/releases) by their tags.
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+ 
+    git checkout tags/<tag_number>
+    ``` 
+1. Follow the setup instructions in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py#L109) 
+to install Forseti server and all the necessary components like Fluentd and Cloud SQL Proxy.
+1. Create firewall rules
+    ```bash
+    # Note: You will need the email address of your service account, you can get the email address by running command
+    # gcloud iam service-accounts list
+    # The email address will be in this format: forseti-server-gcp-#######@PROJECT-ID.iam.gserviceaccount.com
+    
+    # Create firewall rule to block out all the ingress traffic.
+    gcloud compute firewall-rules create forseti-server-deny-all --action DENY --target-service-accounts <SERVICE_ACCOUNT_EMAIL_ADDRESS> --priority 1 --direction INGRESS --rules icmp,udp,tcp
+    
+    # Create firewall rule to open only port tcp:50051 within the internal network (ip-ranges - 10.128.0.0/9).
+    gcloud compute firewall-rules create forseti-server-allow-grpc --action ALLOW --target-service-accounts <SERVICE_ACCOUNT_EMAIL_ADDRESS> --priority 0 --direction INGRESS --rules tcp:50051 --source-ranges 10.128.0.0/9
+    
+    # Create firewall rule to open only port tcp:22 (ssh) to all the external traffics from the internet.
+    gcloud compute firewall-rules create forseti-server-allow-ssh-external --action ALLOW --target-service-accounts <SERVICE_ACCOUNT_EMAIL_ADDRESS> --priority 0 --direction INGRESS --rules tcp:22 --source-ranges 0.0.0.0/0
+    ```
+
+### Configuration
+
+Forseti server configuration and rule files are configured and stored in
+Cloud Storage. For more information, see
+[Configuring Forseti]({% link _docs/latest/howto/configure/configuring-forseti.md %}).
+
+### G Suite Integration
+
+To integrate G Suite, you'll need to enable domain-wide delegation in G Suite.
+For more information, see [Enabling G Suite Google Groups Collection]({% link _docs/latest/configure/gsuite.md %})
+
+## Create a Database
+
+### Create a Cloud SQL instance.
+
+To create a For information about the specifications needed for a Cloud
```

Comment:
We're either missing something, or there are extra words here. "To create a Cloud SQL instance, see the..." ?

Reply:
Updated